### ISSUE TITLE
Training process stop when it comes to validate.May i know why?Thank you for your time

### ISSUE THREAD

**Issue by william3863-prog** [ZJULearning/resa#14]

**Comment by william3863-prog** [comment]
[
![image](https://user-images.githubusercontent.com/83808151/119240327-671c1100-bb81-11eb-9519-0e7954144bf7.png)
](url)
The dataset structure is arranged according to the instruction.

**Comment by william3863-prog** [comment]
for this i just clone the training set from tusimple,do i have to clone the test set together?

**Comment by Turoad** [comment]
You need download the test set.

**Comment by william3863-prog** [comment]
[
![image](https://user-images.githubusercontent.com/83808151/119386618-60fd7000-bcfa-11eb-9eb2-329b55485407.png)
](url)
is it posssible i can change this dataset path?because i locate the dataset at google drive and mount it into my colab notebook as the file is large

**Comment by Turoad** [comment]
you can change the dataset path in the `culane.py` or `tusimple.py`.

**Comment by william3863-prog** [comment]
[
![image](https://user-images.githubusercontent.com/83808151/119539247-8e135680-bdbe-11eb-89c5-bce04fb23494.png)
](url)
i have downloaded the test set and arrange it as shown at the left,i still get this error because some image in the test file collapse?

> You need download the test set.

**Comment by william3863-prog** [comment]
Thank You!I run the training succesfully.How can I view the graph of accuracy,loss and some other data?

**Comment by Turoad** [comment]
Sorry，no graph，log is enough.
